src/21-derive_research_microdata.R

```diff
@@ -82,44 +84,71 @@ fetoinfant <-
     date_of_conception_y %in% c(1989, 1999, 2009, 2014)
     ]
 
+# delete space after icd code
+
+fetoinfant <-
+  fetoinfant[, 
+             cod_icd10 := gsub(" ", "", cod_icd10)]
```

Comment:
Data table `:=` does "in place modification", so no need for a re-assignment using `<-`.